scripts/tools/change_checker.rb

```diff
@@ -0,0 +1,38 @@
+module ChangeChecker
+  def self.changelist(path, recheck=nil)
+    FileUtils.mkdir_p Paths::CHECKLISTS
+  	system ('find ' + path + ' ' + ChangeChecker.prune_paths(path) + ' -type f -exec md5sum {} \; > ' + ChangeChecker.checklist_file(path, recheck))
```

Comment:
Are we sure, it will work on windows? Same for `diff`, `ws` etc?

Reply:
what about https://github.com/filewatcher/filewatcher?

Reply:
Hi Dmitry, the changes should now work for all the operating systems as we are not introducing any external dependencies/tools and using plain Ruby for checking changes in the individual elements.